node/src/actors/chain_manager/handlers.rs

```diff
@@ -1270,9 +1270,9 @@ impl Handler<GetDataRequestReport> for ChainManager {
         } else {
             let dr_pointer_string = format!("DR-REPORT-{}", dr_pointer);
             // Otherwise, try to get it from storage
-            let fut = storage_mngr::get::<_, DataRequestReport>(&dr_pointer_string).and_then(
-                move |dr_report| match dr_report {
-                    Some(x) => futures::finished(DataRequestInfo::from(x)),
+            let fut = storage_mngr::get::<_, DataRequestInfo>(&dr_pointer_string).and_then(
```

Comment:
Changing the type of values from the storage is a breaking change, so any nodes that update to this version will have to delete the storage and synchronize again. We technically could write migrations that convert old entries to the new format, but let's just break it.

Reply:
I did remove the TryFrom conversion to convert DataRequestReport into DataRequestInfo as that was not needed anymore if I replaced it completely. But, yes, I'd say that if this is merged into master, it is for a completely new release (be it a new testnet or mainnet), so people will resynchronize anyway.